crates/pipeline_manager/src/db/pipeline.rs

```diff
@@ -366,11 +367,17 @@ impl PipelineRevision {
         // This unwrap() is ok since we checked above that the program has a schema
         let schema = program.schema.as_ref().unwrap();
 
-        let tables = HashSet::<_>::from_iter(schema.inputs.iter().map(|r| r.name.clone()));
+        let tables = HashSet::<_>::from_iter(schema.inputs.iter().map(|r| {
+            if r.case_sensitive {
```

Comment:
maybe this should be a helper function

Reply:
I agree. This should be handled inside a method like `r.canonical_name()`, rather than clients doing the adjustment.